content/articles/email-forwarding.markdown

```diff
@@ -14,7 +14,7 @@ categories:
 
 ---
 
-Email forwarding can be enabled on a domain-by-domain basis. The cost is $2 per month per domain. The number of messages per month is based on your subscription type:
+Email forwarding can be enabled on a domain-by-domain basis. The cost is $2 per month (or $12 per year if you are on a yearly plan) per domain. The number of messages per month is based on your subscription type:
```

Comment:
Careful there: it's 2*12 :point_right: $24.00 per year

Reply:
Good catch, I can't math :sweat_smile: